Platforms/TrackService.js

```diff
@@ -0,0 +1,38 @@
+const parse = require('csv-parse');
+const fs = require('fs');
+const Track = require('./Track.js');
+require.extensions['.csv'] = function (module, filename) {
+    module.exports = fs.readFileSync(filename, 'utf8');
+};
+const tracksFile = require("./DBSuS-Bahnsteigdaten-Stand2017-04.csv");
+
+class TrackService {
+	
+	constructor() {
+		console.log("construct it")
+		this.tracks = new Promise(function(resolve) {
+			parse(tracksFile, {comment: '#', delimiter: ";", columns: true}, function(err, result) {
```

Comment:
Why do you parse the data on Service creation (Constructor) and not at a specific request?